Native_client python buildings failed

I am following the https://github.com/mozilla/DeepSpeech/blob/master/native_client/README.md, but failed in the last step "Python Buildings". I am running on Mac OS X 10.10.5.
After run 
`PREFIX=/usr/local sudo make install`
I got:
```
(virlen) bogon:native_client gardenia$ PREFIX=/usr/local sudo make install
install -d /usr/local/lib
install -s -m 0644 ../../tensorflow/bazel-bin/tensorflow/libtensorflow.so /usr/local/lib/
/Library/Developer/CommandLineTools/usr/bin/strip: symbols referenced by indirect symbol table entries that can't be stripped in: /usr/local/lib/libtensorflow.so
_OSAtomicAdd64
_OSAtomicCompareAndSwap64
_OSAtomicCompareAndSwap64Barrier
_OSMemoryBarrier
__NSGetExecutablePath
__Unwind_Resume
___bzero
..........
__ZTIc
__ZTIh
__ZTVNSt3__112bad_weak_ptrE
__ZTVNSt3__19basic_iosIcNS_11char_traitsIcEEEE
___gxx_personality_v0
___stack_chk_guard
___stderrp
install: child process failed: xcrun strip - /usr/local/lib//libtensorflow.so
make: *** [install] Error 70
```
There is no errors in previous building steps, and running client via Makefile works fine. It seems to be an issue on Mac OS. Unfortunalely I still don't know how to solve it, any help?

--- a/native_client/Makefile
+++ b/native_client/Makefile
@@ -58,9 +58,9 @@ debug: deepspeech
 
 install: deepspeech
 	install -d ${PREFIX}/lib
-	install -s -m 0644 ${TFDIR}/bazel-bin/tensorflow/libtensorflow.so ${PREFIX}/lib/
-	install -s -m 0644 ${TFDIR}/bazel-bin/native_client/libdeepspeech.so ${PREFIX}/lib/
-	install -s -m 0644 ${TFDIR}/bazel-bin/native_client/libdeepspeech_utils.so ${PREFIX}/lib/
+	install -m 0644 ${TFDIR}/bazel-bin/tensorflow/libtensorflow.so ${PREFIX}/lib/
+	install -m 0644 ${TFDIR}/bazel-bin/native_client/libdeepspeech.so ${PREFIX}/lib/
+	install -m 0644 ${TFDIR}/bazel-bin/native_client/libdeepspeech_utils.so ${PREFIX}/lib/
 	install -d ${PREFIX}/bin
 	install -m 0755 deepspeech ${PREFIX}/bin/
 

